client/src/components/RelatedItems/Outfits/Outfits.jsx

```diff
@@ -0,0 +1,148 @@
+    localStorage.removeItem('slide');
+  };
+
+  const handleAddOutfit = async () => {
+    const allRes = await Promise.all([
+      getDetailById(37311),
+      getReviewById(37311),
+    ]);
+    const finalRes = [];
+
+    const goodsDetail = allRes[0].results[0];
+    let sum = 0;
+    allRes[1].results.forEach((item) => {
+      sum += item.rating;
+    });
+
+    let averageRate = sum / 5;
+    finalRes.push({
+      id: allRes[0].product_id,
+      ...goodsDetail,
+      averageRate,
+    });
+    setSlides(finalRes);
+   
+    localStorage.slide = JSON.stringify(finalRes);
+  };
+
+  return (
+    <>
+      <h4>YOUR OUTFIT</h4>
+      <Swiper
+        modules={[Virtual, Navigation]}
+        onSwiper={setSwiperRef}
+        slidesPerView={3}
+        spaceBetween={50}
+        pagination={{
+          type: 'fraction',
+        }}
+        navigation={true}
+        virtual
+      >
+        <SwiperSlide>
+          <div
+            style={{
+              cursor: 'pointer',
+              height: '380px',
+              width: '250px',
+              border: 'solid',
+              backgroundColor: '#e9e9e9',
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
+            }}
+            onClick={handleAddOutfit}
+          >
+            <PlusOutlined style={{ fontSize: '200px', color: 'grey' }} />
+          </div>
+        </SwiperSlide>
+
+        {slides.map((slideContent, index) => (
+          <SwiperSlide key={slideContent} virtualIndex={index}>
+            <div
+              style={{
+                height: '380px',
+                width: '250px',
+                border: 'solid',
+                display: 'flex',
+                flexDirection: 'column',
+              }}
+            >
+              <div
+                style={{
+                  height: '250px',
+                  backgroundColor: '#e9e9e9',
+                  position: 'relative',
+                  overflow: 'hidden',
+                  backgroundImage: `url(${slideContent.photos[0].url})`,
+                  backgroundSize: 'cover',
+                  backgroundRepeat: 'no-repeat',
+                }}
+              >
+                <CloseCircleOutlined
+                  style={{
+                    position: 'absolute',
+                    top: '10px',
+                    right: '10px',
+                    cursor: 'pointer',
+                    color: 'white',
+                  }}
+                  onClick={() => handleRemoveOutfit(slideContent.id)}
+                />
+              </div>
+              <div
+                style={{
+                  flex: 1,
+                  display: 'flex',
+                  flexDirection: 'column',
+                  justifyContent: 'space-evenly',
+                  alignItems: 'flex-start',
+                  padding: '0 10px',
+                }}
+              >
+                <span style={{ fontWeight: 400, fontSize: '15px' }}>
```

Comment:
This should be an h2 to conform to the styles. The fontSize can be set separately to make it smaller but please use REM units. 1rem would work here.